modules/unsupported/geojson-core/src/main/java/org/geotools/data/geojson/PagingFeatureCollection.java

```diff
@@ -51,23 +51,33 @@ public Integer getMatched() {
 
     @Override
     public int size() {
-        if (matched != null) return matched;
-        return super.size();
+        return matched != null ? matched : super.size();
     }
 
     @Override
     public SimpleFeatureIterator features() {
         return new PagingFeatureIterator(first.features(), next);
     }
 
+    protected SimpleFeatureCollection getFirstCollection() {
+        return first;
+    }
+
+    protected ObjectNode getNext() {
+        return next;
+    }
     /**
      * Reads the next feature collection, or return null if there is none. Subclasses can override if they need a
      * different logic for fetching the next page.
      */
     protected SimpleFeatureCollection readNext(ObjectNode next) throws IOException {
-        if (next == null) return null;
+        if (next == null) {
```

Comment:
Could you please back out all these if related changes?

If you don't like this style, do it right:
* Ask on the list about enforcing curly braces for one liners
* Fix the entire code base (hopefully there is some tool to do it)
* Enforce the style with a QA check ([PMD can do it](https://pmd.github.io/pmd/pmd_rules_java_codestyle.html#controlstatementbraces))

Otherwise we'll end up in formatting petty wars between those that want braces everywhere and those that prefer more compact code.

Reply:
Point taken.